common/lib/knft-get-metadata.ts

```diff
@@ -19,18 +31,22 @@ const nftGetContentType = async (nft: NftType): Promise<string> => {
   let contentType = "text";
   if (!(chainId && address && tokenID && url)) return contentType;
 
-  contentType = contentTypes.get(url) || "";
-  if (contentType) return contentType;
+  if (nft.swarm) {
+    contentType = await swarmGetContentType(nft.swarm);
```

Comment:
should not use swarmGetContentType and redownload content

use previous code, that should work for swarm like for ipfs ?

Reply:
Swarm gateway doesn't supports method "HEAD" of fetch()
we be available in new version of swarmbee

Reply:
ok